### PR TITLE
Rwc2023 challenge serve breakfast

### PR DESCRIPTION
**Pull request by PetervDooren**
issue #1325 keeps track of the issues that are still in the code. I suggest to first merge the rwc2023 version and then make smaller PRs to fix these smaller issues.

### CODE REVIEW

**Review by PetervDooren — COMMENTED** [review]
I approve this PR, but I wrote half of it and opened it.

**Comment by PetervDooren** [comment]
I don't think we should re-tune this for Impuls, rather move to a better method.

File: challenge_serve_breakfast/src/challenge_serve_breakfast/navigate_to_and_pick_item.py [hunk 1]
```diff
@@ -52,10 +56,17 @@ def show_image(package_name, path_to_image_in_package):
 
         @cb_interface(outcomes=["done"])
         def _rotate(_):
-            arm.gripper.send_goal("close", timeout=0.)
-            robot.head.look_up()
-            vyaw = 0.5
-            robot.base.force_drive(0, 0, vyaw, PICK_ROTATION / vyaw)
+            arm.gripper.send_goal("close", timeout=0.0)
+
+            # Look to the operator
```
Comment (shanki98) [inline challenge_serve_breakfast/src/challenge_serve_breakfast/navigate_to_and_pick_item.py:61]: Look at the operator standing to the right of the robot!

File: challenge_serve_breakfast/src/challenge_serve_breakfast/navigate_to_and_place_item_on_table.py [hunk 2]
```diff
@@ -164,7 +162,7 @@ def __init__(self, robot, table_id, table_close_navigation_area):
             )
 
 @cb_interface(outcomes=["done"])
-def _publish_item_poses(marker_array_pub, items):
+def _publish_item_poses(user_data, marker_array_pub, items):
```
Comment (shanki98) [inline challenge_serve_breakfast/src/challenge_serve_breakfast/navigate_to_and_place_item_on_table.py:165]: Why is the callback state defined below ?

**Review by shanki98 — APPROVED** [review]

**Pull request merged**